.github/workflows/game-launch.yml

```
@@ -5,6 +5,13 @@ on:
   schedule:
     # Every Monday at 05:00 UTC (after ui-automation.yml at 04:00)
     - cron: "0 5 * * 1"
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 
 permissions: {}
```

actionlint (1.7.12)

[error] key "permissions" is duplicated in "workflow" section. previously defined at line:8,col:1

(syntax-check)